nab/labeler.py

```diff
@@ -99,14 +99,17 @@ def found(t, data):
         raise ValueError("In the label file %s, timestamps are not in "
                          "chronological order." % self.path)
       
-      # Check that windows are distinct (unique, non-overlapping); subsequent
-      # windows can share a common end and start time
-      num_windows = len(self.windows[relativePath])
-      if num_windows > 1:
-        if not all([(self.windows[relativePath][i+1][0]
-                    - self.windows[relativePath][i][1]).total_seconds() >= 0
-                    for i in range(num_windows-1)]):
-          raise ValueError("In the label file %s, windows overlap." % self.path)
+      # Check that windows are distinct (unique, non-overlapping); the end time
+      # of a window can be the same as the start time of the subsequent window.
+      # This check is not for the combined labels file, and the self.path
+      # condition must match the "--destPath" argument in combine_labels.py.
```

Comment:
Merging this and will the relaxed window overlap issue (#58) with a different PR.